package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.0.5",
   "description": "Javascript developer library for interacting with Matic Network",
   "main": "dist/matic.node.js",
-  "types": "dist/index.d.ts",
+  "types": "lib/index.d.ts",
```

Comment:
Do we also need to remove lib from .gitignore?

Reply:
I don't think
`lib` is generated on compile, we have pushed `dist` folder to resolve CDN dependencies